skypy/power_spectrum/_camb.py

```diff
@@ -72,29 +77,31 @@ def camb(wavenumber, redshift, cosmology, A_s, n_s):
                        omch2=cosmology.Odm0 * h2,
                        omk=cosmology.Ok0,
                        TCMB=cosmology.Tcmb0.value,
-                       mnu=np.sum(cosmology.m_nu.value),
+                       mnu=0.,  # for now, need to solve agreement with CLASS
```

Comment:
Keep the neutrino mass here (but use explicit neutrino mass units). Agreement with CLASS is for cosmologies with massless neutrinos.

```suggestion
                       mnu=np.sum(cosmology.m_nu.to_value(u.eV)),
```